npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/registry_finder.rb

```diff
@@ -168,7 +173,8 @@ def yarnrc_registries
 
             registry = Regexp.last_match[:registry].strip.
                        sub(%r{/+$}, "").
-                       sub(%r{^.*?//}, "")
+                       sub(%r{^.*?//}, "").
+                       gsub(/\s+/, "%20")
```

Comment:
Note to myself: check into ruby perf of chaining `sub` calls like this... how are these implemented under the covers? Ie, are we re-looping all the time such that it'd be better to combine these calls? Need to research how Ruby handles this...